### ISSUE TITLE
Adding a container registry from Dashboard User preferences doesn't work

### ISSUE THREAD

**Issue by l0rd** [eclipse/che#21589]
### Describe the bug

In Dashboard user preference it's possible to add a container registry:

<img width="1105" alt="image" src="https://user-images.githubusercontent.com/606959/181199230-44973cc9-4839-4ebe-b429-a5577bcaf014.png">

But then:

**Starting a workspace that use a private image fails**:
<img width="869" alt="image" src="https://user-images.githubusercontent.com/606959/181200928-07461d0c-6c3c-4126-9916-cfb1fa8bcc3c.png">

**The message on the dashboard doesn't provide the right information**:
<img width="952" alt="image" src="https://user-images.githubusercontent.com/606959/181201549-5f0cba77-4eb0-41cc-802c-634e0ab645fd.png">

**Running `podman pull` of private image fails**:
<img width="1070" alt="image" src="https://user-images.githubusercontent.com/606959/181201845-f05dc9b6-9eb5-4265-891e-e51b1e145fba.png">

### Che version

next (development version)

### Steps to reproduce

1. Add your container registry credentials from the dashboard
2. Try to start a workspace using a devfile that points to a private image
3. Start a workspace where podman works in rootless mode and try to run `podman pull` on private registry

### Expected behavior

If the user has provided added the container registry credentials in his profile:
- [ ] starting a workspace that uses a private container from that registry should work
- [ ] running `podman pull <private image>` from within a workspace should work

If the user has NOT added the container registryin his profile:
- [ ] When the workspace start and fails with `authorization failure` we should report it in the error message and suggest the user to add the container registry credentials in his user profile

### Runtime

OpenShift

### Installation method

OperatorHub

### Environment

Dogfooding instance

### Release Notes Text

If a user added a container registry credential in Eclipse Che User Dashboard, that credential was not used to pull images from the registry resulting in workspaces failing to start.

**Comment by l0rd** [comment]
The generated `devworkspace-container-registry-dockercfg` secret of type `kubernetes.io/dockerconfigjson` should have [the following labels](https://github.com/devfile/devworkspace-operator/blob/main/docs/additional-configuration.adoc#adding-image-pull-secrets-to-workspaces
):

```yaml
    controller.devfile.io/devworkspace_pullsecret: 'true'
    controller.devfile.io/watch-secret: 'true'
```

but it only has `controller.devfile.io/devworkspace_pullsecret: 'true'`.

If adding the label it just work fine.

**Comment by l0rd** [comment]
When I add

```
    controller.devfile.io/devworkspace_pullsecret: 'true'
    controller.devfile.io/watch-secret: 'true'
```

that's the devworkspace status:

```
$ kubectl get dw --watch tilt-n-quarkus
NAME             DEVWORKSPACE ID             PHASE     INFO
tilt-n-quarkus   workspace2b77a9b9659c439c   Stopped   Stopped
tilt-n-quarkus   workspace2b77a9b9659c439c   Stopped   Stopped
tilt-n-quarkus   workspace2b77a9b9659c439c   Starting   Initializing DevWorkspace
tilt-n-quarkus   workspace2b77a9b9659c439c   Starting   Preparing networking
tilt-n-quarkus   workspace2b77a9b9659c439c   Starting   Preparing services
tilt-n-quarkus   workspace2b77a9b9659c439c   Starting   Preparing routes
tilt-n-quarkus   workspace2b77a9b9659c439c   Starting   Preparing routes
tilt-n-quarkus   workspace2b77a9b9659c439c   Starting   Networking ready
tilt-n-quarkus   workspace2b77a9b9659c439c   Starting   Waiting for DevWorkspace ServiceAccount
tilt-n-quarkus   workspace2b77a9b9659c439c   Starting   Waiting for DevWorkspace ServiceAccount
tilt-n-quarkus   workspace2b77a9b9659c439c   Starting   Waiting for workspace deployment
tilt-n-quarkus   workspace2b77a9b9659c439c   Starting   Waiting for workspace deployment
tilt-n-quarkus   workspace2b77a9b9659c439c   Failing    devworkspace failed to progress past phase 'Starting' for longer than timeout (5m)
```

and the events:

```
LAST SEEN   TYPE     REASON              OBJECT                                 MESSAGE
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220   Scaled up replica set workspacec2898ce07e904220-5bcd996745 to 1
0s          Normal   SuccessfulCreate    replicaset/workspacec2898ce07e904220-5bcd996745   Created pod: workspacec2898ce07e904220-5bcd996745-m2sxr
0s          Normal   Scheduled           pod/workspacec2898ce07e904220-5bcd996745-m2sxr    Successfully assigned l0rd-che/workspacec2898ce07e904220-5bcd996745-m2sxr to ip-10-0-142-173.ec2.internal
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled down replica set workspacec2898ce07e904220-5bcd996745 to 0
0s          Normal   SuccessfulDelete    replicaset/workspacec2898ce07e904220-5bcd996745   Deleted pod: workspacec2898ce07e904220-5bcd996745-m2sxr
0s          Normal   AddedInterface      pod/workspacec2898ce07e904220-5bcd996745-m2sxr    Add eth0 [10.129.5.216/23] from openshift-sdn
0s          Normal   Pulling             pod/workspacec2898ce07e904220-5bcd996745-m2sxr    Pulling image "quay.io/che-incubator/che-code:insiders"
0s          Normal   Pulled              pod/workspacec2898ce07e904220-5bcd996745-m2sxr    Successfully pulled image "quay.io/che-incubator/che-code:insiders" in 391.364114ms
0s          Normal   Created             pod/workspacec2898ce07e904220-5bcd996745-m2sxr    Created container che-code-injector
0s          Normal   Started             pod/workspacec2898ce07e904220-5bcd996745-m2sxr    Started container che-code-injector
0s          Normal   Killing             pod/workspacec2898ce07e904220-5bcd996745-m2sxr    Stopping container che-code-injector
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled up replica set workspacec2898ce07e904220-5bcd996745 to 1
0s          Normal   SuccessfulCreate    replicaset/workspacec2898ce07e904220-5bcd996745   Created pod: workspacec2898ce07e904220-5bcd996745-t8chq
0s          Normal   Scheduled           pod/workspacec2898ce07e904220-5bcd996745-t8chq    Successfully assigned l0rd-che/workspacec2898ce07e904220-5bcd996745-t8chq to ip-10-0-139-165.ec2.internal
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled down replica set workspacec2898ce07e904220-5bcd996745 to 0
0s          Normal   SuccessfulDelete    replicaset/workspacec2898ce07e904220-5bcd996745   Deleted pod: workspacec2898ce07e904220-5bcd996745-t8chq
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled up replica set workspacec2898ce07e904220-5bcd996745 to 1
0s          Normal   SuccessfulCreate    replicaset/workspacec2898ce07e904220-5bcd996745   Created pod: workspacec2898ce07e904220-5bcd996745-5zbv7
0s          Normal   Scheduled           pod/workspacec2898ce07e904220-5bcd996745-5zbv7    Successfully assigned l0rd-che/workspacec2898ce07e904220-5bcd996745-5zbv7 to ip-10-0-142-173.ec2.internal
1s          Normal   AddedInterface      pod/workspacec2898ce07e904220-5bcd996745-t8chq    Add eth0 [10.128.8.117/23] from openshift-sdn
0s          Normal   Pulling             pod/workspacec2898ce07e904220-5bcd996745-t8chq    Pulling image "quay.io/che-incubator/che-code:insiders"
0s          Normal   Pulled              pod/workspacec2898ce07e904220-5bcd996745-t8chq    Successfully pulled image "quay.io/che-incubator/che-code:insiders" in 385.084742ms
0s          Normal   AddedInterface      pod/workspacec2898ce07e904220-5bcd996745-5zbv7    Add eth0 [10.129.5.218/23] from openshift-sdn
0s          Normal   Pulling             pod/workspacec2898ce07e904220-5bcd996745-5zbv7    Pulling image "quay.io/che-incubator/che-code:insiders"
0s          Normal   Created             pod/workspacec2898ce07e904220-5bcd996745-t8chq    Created container che-code-injector
0s          Normal   Started             pod/workspacec2898ce07e904220-5bcd996745-t8chq    Started container che-code-injector
0s          Normal   Pulled              pod/workspacec2898ce07e904220-5bcd996745-5zbv7    Successfully pulled image "quay.io/che-incubator/che-code:insiders" in 402.245937ms
0s          Normal   Created             pod/workspacec2898ce07e904220-5bcd996745-5zbv7    Created container che-code-injector
0s          Normal   Started             pod/workspacec2898ce07e904220-5bcd996745-5zbv7    Started container che-code-injector
0s          Normal   Killing             pod/workspacec2898ce07e904220-5bcd996745-t8chq    Stopping container che-code-injector
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled down replica set workspacec2898ce07e904220-5bcd996745 to 0
0s          Normal   SuccessfulDelete    replicaset/workspacec2898ce07e904220-5bcd996745   Deleted pod: workspacec2898ce07e904220-5bcd996745-5zbv7
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled up replica set workspacec2898ce07e904220-5bcd996745 to 1
0s          Normal   SuccessfulCreate    replicaset/workspacec2898ce07e904220-5bcd996745   Created pod: workspacec2898ce07e904220-5bcd996745-zfnxc
0s          Normal   Scheduled           pod/workspacec2898ce07e904220-5bcd996745-zfnxc    Successfully assigned l0rd-che/workspacec2898ce07e904220-5bcd996745-zfnxc to ip-10-0-139-165.ec2.internal
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled down replica set workspacec2898ce07e904220-5bcd996745 to 0
0s          Normal   SuccessfulDelete    replicaset/workspacec2898ce07e904220-5bcd996745   Deleted pod: workspacec2898ce07e904220-5bcd996745-zfnxc
0s          Normal   Killing             pod/workspacec2898ce07e904220-5bcd996745-5zbv7    Stopping container che-code-injector
0s          Normal   AddedInterface      pod/workspacec2898ce07e904220-5bcd996745-zfnxc    Add eth0 [10.128.8.118/23] from openshift-sdn
0s          Normal   Pulling             pod/workspacec2898ce07e904220-5bcd996745-zfnxc    Pulling image "quay.io/che-incubator/che-code:insiders"
0s          Normal   Pulled              pod/workspacec2898ce07e904220-5bcd996745-zfnxc    Successfully pulled image "quay.io/che-incubator/che-code:insiders" in 427.611493ms
0s          Normal   Created             pod/workspacec2898ce07e904220-5bcd996745-zfnxc    Created container che-code-injector
0s          Normal   Started             pod/workspacec2898ce07e904220-5bcd996745-zfnxc    Started container che-code-injector
0s          Normal   Killing             pod/workspacec2898ce07e904220-5bcd996745-zfnxc    Stopping container che-code-injector
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled up replica set workspacec2898ce07e904220-58dcf4d54c to 1
0s          Normal   SuccessfulCreate    replicaset/workspacec2898ce07e904220-58dcf4d54c   Created pod: workspacec2898ce07e904220-58dcf4d54c-gwxxh
0s          Normal   Scheduled           pod/workspacec2898ce07e904220-58dcf4d54c-gwxxh    Successfully assigned l0rd-che/workspacec2898ce07e904220-58dcf4d54c-gwxxh to ip-10-0-142-173.ec2.internal
0s          Normal   AddedInterface      pod/workspacec2898ce07e904220-58dcf4d54c-gwxxh    Add eth0 [10.129.5.221/23] from openshift-sdn
0s          Normal   Pulling             pod/workspacec2898ce07e904220-58dcf4d54c-gwxxh    Pulling image "quay.io/che-incubator/che-code:insiders"
0s          Normal   Pulled              pod/workspacec2898ce07e904220-58dcf4d54c-gwxxh    Successfully pulled image "quay.io/che-incubator/che-code:insiders" in 417.010293ms
0s          Normal   Created             pod/workspacec2898ce07e904220-58dcf4d54c-gwxxh    Created container che-code-injector
0s          Normal   Started             pod/workspacec2898ce07e904220-58dcf4d54c-gwxxh    Started container che-code-injector
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled down replica set workspacec2898ce07e904220-58dcf4d54c to 0
0s          Normal   SuccessfulDelete    replicaset/workspacec2898ce07e904220-58dcf4d54c   Deleted pod: workspacec2898ce07e904220-58dcf4d54c-gwxxh
0s          Normal   Killing             pod/workspacec2898ce07e904220-58dcf4d54c-gwxxh    Stopping container che-code-injector
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled up replica set workspacec2898ce07e904220-98b59b599 to 1
0s          Normal   SuccessfulCreate    replicaset/workspacec2898ce07e904220-98b59b599    Created pod: workspacec2898ce07e904220-98b59b599-kcs6q
0s          Normal   Scheduled           pod/workspacec2898ce07e904220-98b59b599-kcs6q     Successfully assigned l0rd-che/workspacec2898ce07e904220-98b59b599-kcs6q to ip-10-0-142-173.ec2.internal
0s          Normal   AddedInterface      pod/workspacec2898ce07e904220-98b59b599-kcs6q     Add eth0 [10.129.5.223/23] from openshift-sdn
0s          Normal   Pulling             pod/workspacec2898ce07e904220-98b59b599-kcs6q     Pulling image "quay.io/che-incubator/che-code:insiders"
0s          Normal   Pulled              pod/workspacec2898ce07e904220-98b59b599-kcs6q     Successfully pulled image "quay.io/che-incubator/che-code:insiders" in 383.901317ms
0s          Normal   Created             pod/workspacec2898ce07e904220-98b59b599-kcs6q     Created container che-code-injector
0s          Normal   Started             pod/workspacec2898ce07e904220-98b59b599-kcs6q     Started container che-code-injector
0s          Normal   Pulling             pod/workspacec2898ce07e904220-98b59b599-kcs6q     Pulling image "quay.io/devfile/project-clone:sha-45b3501"
0s          Normal   Pulled              pod/workspacec2898ce07e904220-98b59b599-kcs6q     Successfully pulled image "quay.io/devfile/project-clone:sha-45b3501" in 381.208134ms
0s          Normal   Created             pod/workspacec2898ce07e904220-98b59b599-kcs6q     Created container project-clone
0s          Normal   Started             pod/workspacec2898ce07e904220-98b59b599-kcs6q     Started container project-clone
0s          Normal   Pulling             pod/workspacec2898ce07e904220-98b59b599-kcs6q     Pulling image "quay.io/mloriedo/universal-developer-image:tilt"
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled down replica set workspacec2898ce07e904220-98b59b599 to 0
0s          Normal   SuccessfulDelete    replicaset/workspacec2898ce07e904220-98b59b599    Deleted pod: workspacec2898ce07e904220-98b59b599-kcs6q
0s          Normal   Pulled              pod/workspacec2898ce07e904220-98b59b599-kcs6q     Successfully pulled image "quay.io/mloriedo/universal-developer-image:tilt" in 391.831421ms
0s          Normal   Created             pod/workspacec2898ce07e904220-98b59b599-kcs6q     Created container tilt-container
0s          Normal   Started             pod/workspacec2898ce07e904220-98b59b599-kcs6q     Started container tilt-container
0s          Normal   Pulled              pod/workspacec2898ce07e904220-98b59b599-kcs6q     Container image "quay.io/eclipse/che--traefik:v2.8.1-4e52a5e2495484f5e19a49edfd2f652b0bce7b3603fa0df545ed90168ffae1c3" already present on machine
0s          Normal   Created             pod/workspacec2898ce07e904220-98b59b599-kcs6q     Created container che-gateway
0s          Normal   Started             pod/workspacec2898ce07e904220-98b59b599-kcs6q     Started container che-gateway
0s          Normal   Killing             pod/workspacec2898ce07e904220-98b59b599-kcs6q     Stopping container tilt-container
0s          Normal   Killing             pod/workspacec2898ce07e904220-98b59b599-kcs6q     Stopping container che-gateway
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled up replica set workspacec2898ce07e904220-58dcf4d54c to 1
0s          Normal   SuccessfulCreate    replicaset/workspacec2898ce07e904220-58dcf4d54c   Created pod: workspacec2898ce07e904220-58dcf4d54c-pnglz
0s          Normal   Scheduled           pod/workspacec2898ce07e904220-58dcf4d54c-pnglz    Successfully assigned l0rd-che/workspacec2898ce07e904220-58dcf4d54c-pnglz to ip-10-0-142-173.ec2.internal
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled down replica set workspacec2898ce07e904220-58dcf4d54c to 0
0s          Normal   SuccessfulDelete    replicaset/workspacec2898ce07e904220-58dcf4d54c   Deleted pod: workspacec2898ce07e904220-58dcf4d54c-pnglz
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled up replica set workspacec2898ce07e904220-58dcf4d54c to 1
0s          Normal   SuccessfulCreate    replicaset/workspacec2898ce07e904220-58dcf4d54c   Created pod: workspacec2898ce07e904220-58dcf4d54c-kgqb4
0s          Normal   Scheduled           pod/workspacec2898ce07e904220-58dcf4d54c-kgqb4    Successfully assigned l0rd-che/workspacec2898ce07e904220-58dcf4d54c-kgqb4 to ip-10-0-142-173.ec2.internal
0s          Normal   SuccessfulDelete    replicaset/workspacec2898ce07e904220-58dcf4d54c   Deleted pod: workspacec2898ce07e904220-58dcf4d54c-kgqb4
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled down replica set workspacec2898ce07e904220-58dcf4d54c to 0
0s          Normal   AddedInterface      pod/workspacec2898ce07e904220-58dcf4d54c-kgqb4    Add eth0 [10.129.5.225/23] from openshift-sdn
0s          Normal   Pulling             pod/workspacec2898ce07e904220-58dcf4d54c-kgqb4    Pulling image "quay.io/che-incubator/che-code:insiders"
0s          Normal   AddedInterface      pod/workspacec2898ce07e904220-58dcf4d54c-pnglz    Add eth0 [10.129.5.224/23] from openshift-sdn
0s          Normal   Pulling             pod/workspacec2898ce07e904220-58dcf4d54c-pnglz    Pulling image "quay.io/che-incubator/che-code:insiders"
0s          Normal   Pulled              pod/workspacec2898ce07e904220-58dcf4d54c-kgqb4    Successfully pulled image "quay.io/che-incubator/che-code:insiders" in 410.058489ms
0s          Normal   Pulled              pod/workspacec2898ce07e904220-58dcf4d54c-pnglz    Successfully pulled image "quay.io/che-incubator/che-code:insiders" in 400.949239ms
0s          Normal   Created             pod/workspacec2898ce07e904220-58dcf4d54c-kgqb4    Created container che-code-injector
0s          Normal   Started             pod/workspacec2898ce07e904220-58dcf4d54c-kgqb4    Started container che-code-injector
0s          Normal   Killing             pod/workspacec2898ce07e904220-58dcf4d54c-kgqb4    Stopping container che-code-injector
0s          Normal   Created             pod/workspacec2898ce07e904220-58dcf4d54c-pnglz    Created container che-code-injector
0s          Normal   Started             pod/workspacec2898ce07e904220-58dcf4d54c-pnglz    Started container che-code-injector
0s          Normal   Killing             pod/workspacec2898ce07e904220-58dcf4d54c-pnglz    Stopping container che-code-injector
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled up replica set workspacec2898ce07e904220-6cc4b4d557 to 1
0s          Normal   SuccessfulCreate    replicaset/workspacec2898ce07e904220-6cc4b4d557   Created pod: workspacec2898ce07e904220-6cc4b4d557-56v66
0s          Normal   Scheduled           pod/workspacec2898ce07e904220-6cc4b4d557-56v66    Successfully assigned l0rd-che/workspacec2898ce07e904220-6cc4b4d557-56v66 to ip-10-0-142-173.ec2.internal
0s          Normal   SuccessfulDelete    replicaset/workspacec2898ce07e904220-6cc4b4d557   Deleted pod: workspacec2898ce07e904220-6cc4b4d557-56v66
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled down replica set workspacec2898ce07e904220-6cc4b4d557 to 0
0s          Normal   AddedInterface      pod/workspacec2898ce07e904220-6cc4b4d557-56v66    Add eth0 [10.129.5.226/23] from openshift-sdn
0s          Normal   Pulling             pod/workspacec2898ce07e904220-6cc4b4d557-56v66    Pulling image "quay.io/che-incubator/che-code:insiders"
0s          Normal   Pulled              pod/workspacec2898ce07e904220-6cc4b4d557-56v66    Successfully pulled image "quay.io/che-incubator/che-code:insiders" in 418.71749ms
0s          Normal   Created             pod/workspacec2898ce07e904220-6cc4b4d557-56v66    Created container che-code-injector
0s          Normal   Started             pod/workspacec2898ce07e904220-6cc4b4d557-56v66    Started container che-code-injector
0s          Normal   Killing             pod/workspacec2898ce07e904220-6cc4b4d557-56v66    Stopping container che-code-injector
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled up replica set workspacec2898ce07e904220-58dcf4d54c to 1
0s          Normal   SuccessfulCreate    replicaset/workspacec2898ce07e904220-58dcf4d54c   Created pod: workspacec2898ce07e904220-58dcf4d54c-mlmqq
0s          Normal   Scheduled           pod/workspacec2898ce07e904220-58dcf4d54c-mlmqq    Successfully assigned l0rd-che/workspacec2898ce07e904220-58dcf4d54c-mlmqq to ip-10-0-142-173.ec2.internal
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled down replica set workspacec2898ce07e904220-58dcf4d54c to 0
0s          Normal   SuccessfulDelete    replicaset/workspacec2898ce07e904220-58dcf4d54c   Deleted pod: workspacec2898ce07e904220-58dcf4d54c-mlmqq
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled up replica set workspacec2898ce07e904220-58dcf4d54c to 1
0s          Normal   SuccessfulCreate    replicaset/workspacec2898ce07e904220-58dcf4d54c   Created pod: workspacec2898ce07e904220-58dcf4d54c-fjtm5
0s          Normal   Scheduled           pod/workspacec2898ce07e904220-58dcf4d54c-fjtm5    Successfully assigned l0rd-che/workspacec2898ce07e904220-58dcf4d54c-fjtm5 to ip-10-0-142-173.ec2.internal
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled down replica set workspacec2898ce07e904220-58dcf4d54c to 0
0s          Normal   SuccessfulDelete    replicaset/workspacec2898ce07e904220-58dcf4d54c   Deleted pod: workspacec2898ce07e904220-58dcf4d54c-fjtm5
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled up replica set workspacec2898ce07e904220-58dcf4d54c to 1
0s          Normal   SuccessfulCreate    replicaset/workspacec2898ce07e904220-58dcf4d54c   Created pod: workspacec2898ce07e904220-58dcf4d54c-jbgkr
0s          Normal   Scheduled           pod/workspacec2898ce07e904220-58dcf4d54c-jbgkr    Successfully assigned l0rd-che/workspacec2898ce07e904220-58dcf4d54c-jbgkr to ip-10-0-139-165.ec2.internal
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled down replica set workspacec2898ce07e904220-58dcf4d54c to 0
0s          Normal   SuccessfulDelete    replicaset/workspacec2898ce07e904220-58dcf4d54c   Deleted pod: workspacec2898ce07e904220-58dcf4d54c-jbgkr
0s          Normal   AddedInterface      pod/workspacec2898ce07e904220-58dcf4d54c-mlmqq    Add eth0 [10.129.5.227/23] from openshift-sdn
0s          Normal   Pulling             pod/workspacec2898ce07e904220-58dcf4d54c-mlmqq    Pulling image "quay.io/che-incubator/che-code:insiders"
0s          Normal   Pulled              pod/workspacec2898ce07e904220-58dcf4d54c-mlmqq    Successfully pulled image "quay.io/che-incubator/che-code:insiders" in 402.807892ms
0s          Normal   AddedInterface      pod/workspacec2898ce07e904220-58dcf4d54c-fjtm5    Add eth0 [10.129.5.228/23] from openshift-sdn
0s          Normal   Pulling             pod/workspacec2898ce07e904220-58dcf4d54c-fjtm5    Pulling image "quay.io/che-incubator/che-code:insiders"
0s          Normal   Created             pod/workspacec2898ce07e904220-58dcf4d54c-mlmqq    Created container che-code-injector
0s          Normal   Started             pod/workspacec2898ce07e904220-58dcf4d54c-mlmqq    Started container che-code-injector
0s          Normal   ScalingReplicaSet   deployment/workspacec2898ce07e904220              Scaled up replica set workspacec2898ce07e904220-58dcf4d54c to 1
0s          Normal   SuccessfulCreate    replicaset/workspacec2898ce07e904220-58dcf4d54c   Created pod: workspacec2898ce07e904220-58dcf4d54c-lhphh
0s          Normal   Scheduled           pod/workspacec2898ce07e904220-58dcf4d54c-lhphh    Successfully assigned l0rd-che/workspacec2898ce07e904220-58dcf4d54c-lhphh to ip-10-0-142-173.ec2.internal
0s          Normal   Pulled              pod/workspacec2898ce07e904220-58dcf4d54c-fjtm5    Successfully pulled image "quay.io/che-incubator/che-code:insiders" in 429.191927ms
0s          Normal   SuccessfulDelete    replicaset/workspacec2898ce07e904220-58dcf4d54c   Deleted pod: workspacec2898ce07e904220-58dcf4d54c-lhphh
0s          Normal   AddedInterface      pod/workspacec2898ce07e904220-58dcf4d54c-jbgkr    Add eth0 [10.128.8.119/23] from openshift-sdn
0s          Normal   Created             pod/workspacec2898ce07e904220-58dcf4d54c-fjtm5    Created container che-code-injector
0s          Normal   Pulling             pod/workspacec2898ce07e904220-58dcf4d54c-jbgkr    Pulling image "quay.io/che-incubator/che-code:insiders"
0s          Normal   Started             pod/workspacec2898ce07e904220-58dcf4d54c-fjtm5    Started container che-code-injector
0s          Normal   Killing             pod/workspacec2898ce07e904220-58dcf4d54c-mlmqq    Stopping container che-code-injector
0s          Normal   Killing             pod/workspacec2898ce07e904220-58dcf4d54c-fjtm5    Stopping container che-code-injector
0s          Normal   Pulled              pod/workspacec2898ce07e904220-58dcf4d54c-jbgkr    Successfully pulled image "quay.io/che-incubator/che-code:insiders" in 433.00691ms
0s          Normal   Created             pod/workspacec2898ce07e904220-58dcf4d54c-jbgkr    Created container che-code-injector
0s          Normal   Started             pod/workspacec2898ce07e904220-58dcf4d54c-jbgkr    Started container che-code-injector
0s          Normal   Killing             pod/workspacec2898ce07e904220-58dcf4d54c-jbgkr    Stopping container che-code-injector
0s          Normal   AddedInterface      pod/workspacec2898ce07e904220-58dcf4d54c-lhphh    Add eth0 [10.129.5.229/23] from openshift-sdn
0s          Normal   Pulling             pod/workspacec2898ce07e904220-58dcf4d54c-lhphh    Pulling image "quay.io/che-incubator/che-code:insiders"
0s          Normal   Pulled              pod/workspacec2898ce07e904220-58dcf4d54c-lhphh    Successfully pulled image "quay.io/che-incubator/che-code:insiders" in 390.280842ms
0s          Normal   Created             pod/workspacec2898ce07e904220-58dcf4d54c-lhphh    Created container che-code-injector
0s          Normal   Started             pod/workspacec2898ce07e904220-58dcf4d54c-lhphh    Started container che-code-injector
0s          Normal   SuccessfulCreate    replicaset/workspacec2898ce07e904220-58dcf4d54c   Created pod: workspacec2898ce07e904220-58dcf4d54c-jqvd8
0s          Normal   Scheduled           pod/workspacec2898ce07e904220-58dcf4d54c-jqvd8    Successfully assigned l0rd-che/workspacec2898ce07e904220-58dcf4d54c-jqvd8 to ip-10-0-139-165.ec2.internal
0s          Normal   SuccessfulDelete    replicaset/workspacec2898ce07e904220-58dcf4d54c   Deleted pod: workspacec2898ce07e904220-58dcf4d54c-jqvd8
0s          Normal   Killing             pod/workspacec2898ce07e904220-58dcf4d54c-lhphh    Stopping container che-code-injector
0s          Normal   AddedInterface      pod/workspacec2898ce07e904220-58dcf4d54c-jqvd8    Add eth0 [10.128.8.120/23] from openshift-sdn
0s          Normal   Pulling             pod/workspacec2898ce07e904220-58dcf4d54c-jqvd8    Pulling image "quay.io/che-incubator/che-code:insiders"
0s          Normal   Pulled              pod/workspacec2898ce07e904220-58dcf4d54c-jqvd8    Successfully pulled image "quay.io/che-incubator/che-code:insiders" in 450.429174ms
0s          Normal   Created             pod/workspacec2898ce07e904220-58dcf4d54c-jqvd8    Created container che-code-injector
0s          Normal   Started             pod/workspacec2898ce07e904220-58dcf4d54c-jqvd8    Started container che-code-injector
0s          Normal   Killing             pod/workspacec2898ce07e904220-58dcf4d54c-jqvd8    Stopping container che-code-injector
0s          Normal   SuccessfulCreate    replicaset/workspacec2898ce07e904220-58dcf4d54c   Created pod: workspacec2898ce07e904220-58dcf4d54c-w9bm7
0s          Normal   Scheduled           pod/workspacec2898ce07e904220-58dcf4d54c-w9bm7    Successfully assigned l0rd-che/workspacec2898ce07e904220-58dcf4d54c-w9bm7 to ip-10-0-142-173.ec2.internal
0s          Normal   SuccessfulDelete    replicaset/workspacec2898ce07e904220-58dcf4d54c   Deleted pod: workspacec2898ce07e904220-58dcf4d54c-w9bm7
0s          Normal   SuccessfulCreate    replicaset/workspacec2898ce07e904220-58dcf4d54c   (combined from similar events): Created pod: workspacec2898ce07e904220-58dcf4d54c-25qp9
0s          Normal   Scheduled           pod/workspacec2898ce07e904220-58dcf4d54c-25qp9    Successfully assigned l0rd-che/workspacec2898ce07e904220-58dcf4d54c-25qp9 to ip-10-0-142-173.ec2.internal
0s          Normal   AddedInterface      pod/workspacec2898ce07e904220-58dcf4d54c-w9bm7    Add eth0 [10.129.5.230/23] from openshift-sdn
0s          Normal   AddedInterface      pod/workspacec2898ce07e904220-58dcf4d54c-25qp9    Add eth0 [10.129.5.231/23] from openshift-sdn
0s          Normal   Pulling             pod/workspacec2898ce07e904220-58dcf4d54c-w9bm7    Pulling image "quay.io/che-incubator/che-code:insiders"
0s          Normal   Pulling             pod/workspacec2898ce07e904220-58dcf4d54c-25qp9    Pulling image "quay.io/che-incubator/che-code:insiders"
0s          Normal   Pulled              pod/workspacec2898ce07e904220-58dcf4d54c-w9bm7    Successfully pulled image "quay.io/che-incubator/che-code:insiders" in 394.397065ms
0s          Normal   Pulled              pod/workspacec2898ce07e904220-58dcf4d54c-25qp9    Successfully pulled image "quay.io/che-incubator/che-code:insiders" in 408.678497ms
0s          Normal   Created             pod/workspacec2898ce07e904220-58dcf4d54c-w9bm7    Created container che-code-injector
0s          Normal   Started             pod/workspacec2898ce07e904220-58dcf4d54c-w9bm7    Started container che-code-injector
0s          Normal   Created             pod/workspacec2898ce07e904220-58dcf4d54c-25qp9    Created container che-code-injector
0s          Normal   Started             pod/workspacec2898ce07e904220-58dcf4d54c-25qp9    Started container che-code-injector
0s          Normal   Killing             pod/workspacec2898ce07e904220-58dcf4d54c-w9bm7    Stopping container che-code-injector
0s          Normal   Killing             pod/workspacec2898ce07e904220-58dcf4d54c-25qp9    Stopping container che-code-injector
0s          Normal   SuccessfulDelete    replicaset/workspacec2898ce07e904220-58dcf4d54c   (combined from similar events): Deleted pod: workspacec2898ce07e904220-58dcf4d54c-25qp9
```

**Comment by amisevsk** [comment]
Caused by https://github.com/devfile/devworkspace-operator/issues/897.

Fixed by https://github.com/devfile/devworkspace-operator/pull/898

**Comment by l0rd** [comment]
Removing label `area/devworkspace-operator` as, after the [devworkspace operator fix](https://github.com/devfile/devworkspace-operator/pull/898), the problem is dashboard side only.

**Comment by amisevsk** [comment]
Note that the fix lands in v0.15.3 upstream and will be in v0.16.0 downstream; there are no current plans to release v0.15.3 downstream (this can be changed if this issue is deemed important)

**Comment by l0rd** [comment]
Closing this issue as the commit has been backported to 7.52.x now.
